main/updater.js

```diff
@@ -134,28 +134,11 @@ function onUpdateAvailable ({ version /*, releaseNotes */ }) {
   updateAvailable = true
   nextVersion = version
 
-  ipcMain.emit(ipcMainEvents.UPDATE_AVAILABLE)
   log.info(`Update to version ${version} is available, downloading..`)
   autoUpdater.downloadUpdate().then(
     _ => log.info('Update downloaded'),
     err => log.error('Cannot download the update.', err)
   )
-
-  if (!checkingManually) { return }
-  // do not toggle checkingManually off here so we can show a dialog once the
-  // download is finished.
-
-  const buttonIx = showDialogSync({
-    title: 'Update available',
-    message: `A new version ${version} of Filecoin Station is available. ` +
-      'The download will begin shortly in the background.',
-    type: 'info',
-    buttons: ['Close', 'Show Release Notes']
-  })
-
-  if (buttonIx === 1) {
-    openReleaseNotes()
-  }
```

Comment:
IIUC, after your changes, when I trigger a manual check, I will no longer have an easy way to find the release notes for the new version.


The dialog opened by `onUpdateDownloaded` has only these two buttons:
```
 buttons: ['Later', 'Install now']
```

I propose to add the button "Show Release Notes" to `onUpdateDownloaded` dialog.

WDYT?

Reply:
You're right, release notes are now only linked to from the update banner. Added!